lib/new-flatfile.ts

```diff
@@ -72,3 +73,35 @@ export const addGuestToSpace = async ({
     return null;
   }
 };
+
+export const listWorkbooks = async ({ spaceId }: { spaceId: string }) => {
+  try {
+    const flatfile = flatfileClient();
+
+    const result = await flatfile.workbooks.list(
+      spaceId as ListWorkbooksRequest
```

Comment:
This `as` is odd, what is the actual type expected here?

Reply:
I just needed to wrap in object